nss-day-cohort-44/rare-rest-api-fanciful-snowflake
# Ps tag views

# Description
Changed `tags` viewset to match frontend naming conventions (tag to label)

## Type of change

# Testing Instructions
see if tags load with correct name in DOM

1. `git fetch --all`
1. `git checkout ps-tags-view`
1. `serve`
